guardrails/llm_providers.py

```diff
@@ -53,6 +53,14 @@ def openai_wrapper(text: str, *args, **kwargs):
     return openai_response["choices"][0]["text"]
 
 
+def openai_embedding_wrapper(
```

Comment:
nit: Can you move this to a separate module, e.g. `embedding.py`?

Reply:
Probably best to move this inside OpenAIEmbedding.